SECURITY.md

```
@@ -8,7 +8,7 @@ stable release, security fixes will land directly on `main` and be included in
 the next tagged version.
 
 ## Reporting a Vulnerability
-Please email **security@echoengine.dev** with details. Include:
+Please email **security@flyingrobots.dev** with details. Include:
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)